<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLinkEdit</class>
 <widget class="QDialog" name="DialogLinkEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>链接点编辑</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>70</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>链接点代号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdLinkTag">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>5</y>
     <width>180</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>37</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>项目代号</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbProTag">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>33</y>
     <width>180</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>文字位置</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CbTextPos">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>60</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>箭头</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>箭尾</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>94</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdDesc">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>90</y>
     <width>180</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnProSet">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>32</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>123</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>123</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
